### PR TITLE
{bio}[foss/2018b] PHLAT v1.1

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PHLAT/PHLAT-1.1-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    ('pysam', '0.8.3', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/PHLAT/PHLAT-1.1-foss-2018b-Python-2.7.15.eb]: @smoors This is *ancient*, can you clarify this?

**Comment by akesandgren** [comment]
@smoors a build test report please?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic420 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fe7934935e0d8767309eb9417a34aff2 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**